packages/loot-core/src/client/actions/queries.ts

```diff
@@ -205,40 +204,31 @@ export function deleteCategory(id: string, transferId?: string) {
       }
 
       throw new Error(error);
-    } else {
-      dispatch(getCategories());
-      // Also need to refresh payees because they might use one of the
-      // deleted categories as the default category
-      dispatch(getPayees());
```

Comment:
I don't think we need to update payees on category delete. Please correct me if I'm wrong on this one.